charts/harbor-operator/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+1. Get the application URL by running these commands:
```

Comment:
Question: why do we put the chart content here? Any considerations?

Reply:
To be honest, I do not know. what do you suggest?
I though `./charts/harbor-operator` was clear enough.
I am planning to enable github page to serve the chart. WDYT?